### PR TITLE
[PBE-349] Tidy up protos and remove unnecessary and unused fields

### PR DESCRIPTION
**Pull request by su225**
We have many proto fields that are either not needed because they are outdated or very unlikely to be implemented. This PR is for cleaning them up.

**Note for SDK:** Fields are removed instead of just being marked `reserved`. This is because we have not yet released the beta as of now.

### CODE REVIEW

File: protobuf/video/sfu/event/events.proto [hunk 1]
```diff
@@ -83,18 +82,16 @@ message TrackUnpublished {
   models.Participant participant = 5;
 }
 
+// JoinRequest is sent by the participant to join a call. The
+// call ID of the call should be encoded in the auth token.
 message JoinRequest {
   string token = 1;
   string session_id = 2;
-  // dumb SDP that allow us to extract subscriber's decode codecs
-  string subscriber_sdp = 3;
```
Comment (oliverlaz) [inline protobuf/video/sfu/event/events.proto:90]: How would the SFU determine the decode capabilities of the clients if we remove this? 
Did something change in the meantime?

Comment (su225) [reply protobuf/video/sfu/event/events.proto:90]: Well...We currently don't use this field in the SFU. It has been like that for quite some time now. @thesyncim - any plan to use it in the future?

Comment (su225) [reply protobuf/video/sfu/event/events.proto:90]: Spoke to @thesyncim. I'll be restoring the field.

Comment (oliverlaz) [reply protobuf/video/sfu/event/events.proto]: Awesome! Thank you for the clarification 👍

**Pull request closed without merging**